workspaces/jenkins/plugins/jenkins-scaffolder/README.md

```diff
@@ -0,0 +1,64 @@
+# Scaffolder plugin for Jenkins
+
+Welcome to the `plugin-jenkins-scaffolder` custom action!
+
+This contains one action: `jenkins:job:create`
+
+The `jenkins:job:create` action creates a new job in Jenkins.
+
+## New Backend System
+
+The jenkins backend plugin has support for the [new backend system](https://backstage.io/docs/backend-system/), here's how you can set that up:
+
+In your `packages/backend/src/index.ts` make the following changes:
+
+```diff
+  import { createBackend } from '@backstage/backend-defaults';
+  const backend = createBackend();
+  // ... other feature additions
+  backend.add(import('@backstage-community/plugin-jenkins-scaffolder'));
+  backend.start();
+```
+
+### Authorization
+
+In order to use `plugin-jenkins-scaffolder`, you must provide a username and api key to allow access the Jenkins API (permission to create jobs is required)
+
+You must define your jenkins username and api key in the `app-config.yaml`:
+
+```yaml
+scaffolder:
+  jenkins:
+    username: ${JENKINS_USERNAME}
+    key: ${JENKINS_API_KEY}
+    server: ${JENKINS_SERVER_URL}
```

Comment:
perhaps we could reuse the same configuration as defined in the [jenkins plugin](https://github.com/backstage/community-plugins/tree/main/workspaces/jenkins/plugins/jenkins-backend#example---single-global-instance)?

Reply:
Good idea!